<ui version="4.0" >
 <class>KWalletWizardPagePassword</class>
 <widget class="QWidget" name="KWalletWizardPagePassword" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>401</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QLabel" name="textLabel3" >
     <property name="text" >
      <string>Various applications may attempt to use the KDE wallet to store passwords or other information such as web form data and cookies.  If you would like these applications to use the wallet, you must enable it now and choose a password.  The password you choose &lt;i>cannot&lt;/i> be recovered if it is lost, and will allow anyone who knows it to obtain all the information contained in the wallet.</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QCheckBox" name="_useWallet" >
     <property name="text" >
      <string>Yes, I wish to use the KDE wallet to store my personal information.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_3" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Enter a new password:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignAbsolute|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
       </property>
       <property name="buddy" >
        <cstring>_pass1</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel2_3" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Verify password:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignAbsolute|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
       </property>
       <property name="buddy" >
        <cstring>_pass2</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="KLineEdit" name="_pass1" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="_pass2" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>101</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="4" >
    <widget class="QLabel" name="_matchLabel" >
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignAbsolute|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_useWallet</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel1_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>108</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel" >
     <x>217</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_useWallet</sender>
   <signal>toggled(bool)</signal>
   <receiver>_pass1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>455</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel" >
     <x>394</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_useWallet</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel2_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>60</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel" >
     <x>131</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_useWallet</sender>
   <signal>toggled(bool)</signal>
   <receiver>_pass2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>487</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel" >
     <x>371</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_useWallet</sender>
   <signal>clicked()</signal>
   <receiver>_pass1</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel" >
     <x>269</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
